radicle-node/src/worker/fetch.rs

```diff
@@ -124,14 +127,24 @@ impl<'a> StagingPhaseInitial<'a> {
     /// Convert the [`StagingPhaseInitial`] into [`StagingPhaseFinal`] to continue
     /// the fetch process.
     pub fn into_final(self) -> Result<StagingPhaseFinal<'a>, error::Transition> {
-        let remotes = match &self.repo {
+        let trusted = match &self.repo {
             StagedRepository::Cloning(repo) => {
                 log::debug!(target: "worker", "Loading remotes for clone");
                 let oid = ReadRepository::identity_head(repo)?;
                 log::trace!(target: "worker", "Loading 'rad/id' @ {oid}");
                 let (doc, _) = Doc::<Unverified>::load_at(oid, repo)?;
                 let doc = doc.verified()?;
-                doc.delegates.map(PublicKey::from)
+                let mut trusted = match self.namespaces.clone() {
+                    Namespaces::All => HashSet::new(),
+                    // TODO(finto): this is one of those cases where
+                    // having the `One` variant doesn't make any
+                    // sense.
```

Comment:
Should we remove it and just have `Namespaces::All` and `Namespaces::Trusted(HashSet)`?